tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -1132,8 +1132,8 @@ def test_with_wrong_builder_specified_python_runtime(self, use_container):
 
         # runtime is chosen based off current python version.
         runtime = self._get_python_version()
-        # BuildMethod is set to the ruby2.7, this should cause failure.
-        overrides = {"Runtime": runtime, "CodeUri": "Provided", "Handler": "main.handler", "BuildMethod": "ruby2.7"}
+        # BuildMethod is set to the java8, this should cause failure.
+        overrides = {"Runtime": runtime, "CodeUri": "Provided", "Handler": "main.handler", "BuildMethod": "java8"}
```

Comment:
Why did "ruby2.7" not work for this test case?

Reply:
Recently we have released a change to unblock Ruby users to build their project from requiring a manifest file. In this test case, the project is provided a python manifest but this will not fail Ruby builder.

Reply:
got it, thanks!